### PR TITLE
Adds a message if someone runs one of the plugins directly

### PR DESCRIPTION
**Pull request by rtm516**
Adds a message if someone runs one of the plugins directly that tells them that its a plugin version and where to put it.
image

### CODE REVIEW

File: bootstrap/bukkit/src/main/java/org/geysermc/platform/bukkit/GeyserBukkitRunnable.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ *  @author GeyserMC
+ *  @link https://github.com/GeyserMC/Geyser
+ *
+ */
+
+package org.geysermc.platform.bukkit;
+
+import org.geysermc.common.runnable.IGeyserRunnable;
+
+public class GeyserBukkitRunnable extends IGeyserRunnable {
```
Comment (RednedEpic) [inline bootstrap/bukkit/src/main/java/org/geysermc/platform/bukkit/GeyserBukkitRunnable.java:31]: `Runnable` sound more like the scheduler used inside of Bukkit itself, so the naming here may be quite confusing for people. Perhaps use something like `Geyser<platform>Main`.

**Comment by jackson-57** [comment]
you might want to mention the standalone version as well, but this is a good idea. I've also seen a couple people try to double click to run the plugin version, wouldn't the message flash on screen briefly and disappear?

**Comment by rtm516** [comment]
Preview of the message box:
image

**Comment by DoctorMacc** [comment]
Maybe change Bukkit to "Bukkit, Spigot or Paper"?

**Pull request merged**